test/utils/nextpage/sa/partnership/HaveYouRegisteredPartnershipNextPageSpec.scala

```diff
@@ -16,23 +16,74 @@
 
 package utils.nextpage.sa.partnership
 
+import models.requests.{AuthenticatedRequest, ServiceInfoRequest}
 import models.sa.partnership.HaveYouRegisteredPartnership
+import play.api.mvc.AnyContent
+import play.twirl.api.Html
+import uk.gov.hmrc.auth.core.{Enrolment, EnrolmentIdentifier, Enrolments}
+import utils.HmrcEnrolmentType
 import utils.NextPage
 import utils.nextpage.NextPageSpecBase
 
 class HaveYouRegisteredPartnershipNextPageSpec extends NextPageSpecBase {
 
+  val serviceRequest: ServiceInfoRequest[AnyContent] =
+    ServiceInfoRequest(AuthenticatedRequest(request, "", Enrolments(Set()), None), Html(""))
+
+  val saEnrolment = Enrolment(key = HmrcEnrolmentType.SA, identifiers = Seq(), state = "Activated")
```

Comment:
This enrolments stuff is used in a couple of files, could it be refactored out to a common place?